glib:arm64-osx build failure

**Host Environment**
-   OS: [macOS 11.6]
-   Compiler: Apple clang 12.0.5

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install --triplet=arm64-osx openimageio`

**Failure logs** 
```
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:127 (message):
    Command failed: /Library/Frameworks/Python.framework/Versions/3.7/bin/python3 /Users/mark/Projects/github/vcpkg/downloads/tools/meson/meson-aeda7f249c4a5dbbecc52e44f382246a2377b5b0/meson.py -Dbuild_tests=false -Dinstalled_tests=false -Dselinux=disabled -Dinternal_pcre=false --buildtype plain --backend ninja --wrap-mode nodownload --cross /Users/mark/Projects/github/vcpkg/buildtrees/glib/meson-cross-arm64-osx.log --native /Users/mark/Projects/github/vcpkg/scripts/buildsystems/meson/none.txt --default-library static --libdir lib --cross /Users/mark/Projects/github/vcpkg/buildtrees/glib/meson-cross-arm64-osx-debug.log -Ddebug=true --prefix /Users/mark/Projects/github/vcpkg/packages/glib_arm64-osx/debug --includedir ../include -Dcmake_prefix_path=['/Users/mark/Projects/github/vcpkg/installed/arm64-osx/debug','/Users/mark/Projects/github/vcpkg/installed/arm64-osx'] /Users/mark/Projects/github/vcpkg/buildtrees/glib/src/glib-2-b84205c786.clean
    Working Directory: /Users/mark/Projects/github/vcpkg/buildtrees/glib/arm64-osx-dbg
    Error code: 1
    See logs for more information:
      /Users/mark/Projects/github/vcpkg/buildtrees/glib/config-arm64-osx-dbg-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_meson.cmake:512 (vcpkg_execute_required_process)
  ports/glib/portfile.cmake:42 (vcpkg_configure_meson)
  scripts/ports.cmake:140 (include)


Error: Building package glib:arm64-osx failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `./vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: glib:arm64-osx
  Vcpkg version: 2021-09-10-unknownhash

Additionally, attach any relevant sections from the log files above.
```
-Please attach any additional failure logs mentioned in the console output.

[config-x64-osx-dbg-out.log](https://github.com/microsoft/vcpkg/files/7229663/config-x64-osx-dbg-out.log)

The relevant part is

```
Compiler for language objc for the build machine not found.

../src/glib-2-b84205c786.clean/meson.build:733:2: ERROR: 'objc' compiler binary not defined in cross or native file

A full log can be found at /Users/mark/Projects/github/vcpkg/buildtrees/glib/arm64-osx-dbg/meson-logs/meson-log.txt
```

Here is the log file mentioned in that message.

[meson-log.txt](https://github.com/microsoft/vcpkg/files/7229667/meson-log.txt)

The relevant part of it:

```
Compiler for language objc for the build machine not found.

../src/glib-2-b84205c786.clean/meson.build:733:2: ERROR: 'objc' compiler binary not defined in cross or native file
```

doesn't add anything useful.

**Additional context**
- Vcpkg version: 2021-09-10-unknownhash

--- a/ports/glib/portfile.cmake
+++ b/ports/glib/portfile.cmake
@@ -13,6 +13,10 @@ vcpkg_extract_source_archive(SOURCE_PATH
         fix-build-race-on-gio.patch # https://gitlab.gnome.org/GNOME/glib/-/merge_requests/3512
 )
 
+if(APPLE)
+    list(APPEND VCPKG_CMAKE_CONFIGURE_OPTIONS "-DVCPKG_ENABLE_OBJC=1")
+endif()
+
 vcpkg_list(SET OPTIONS)
 if (selinux IN_LIST FEATURES)
     if(NOT EXISTS "/usr/include/selinux")
@@ -38,6 +42,7 @@ endif()
 
 vcpkg_configure_meson(
     SOURCE_PATH "${SOURCE_PATH}"
+    LANGUAGES C CXX OBJC OBJCXX
     ADDITIONAL_BINARIES
         ${ADDITIONAL_BINARIES}
     OPTIONS

--- a/ports/glib/vcpkg.json
+++ b/ports/glib/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "glib",
   "version": "2.76.3",
-  "port-version": 2,
+  "port-version": 3,
   "description": "Portable, general-purpose utility library.",
   "homepage": "https://developer.gnome.org/glib/",
   "license": "LGPL-2.1-or-later",

--- a/scripts/get_cmake_vars/CMakeLists.txt
+++ b/scripts/get_cmake_vars/CMakeLists.txt
@@ -11,6 +11,9 @@ else()
 endif()
 
 set(VCPKG_LANGUAGES "C;CXX" CACHE STRING "Languages to enables for this project")
+if(VCPKG_ENABLE_OBJC)
+    list(APPEND VCPKG_LANGUAGES "OBJC")
+endif()
 
 project(get_cmake_vars LANGUAGES ${VCPKG_LANGUAGES})
 

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2910,7 +2910,7 @@
     },
     "glib": {
       "baseline": "2.76.3",
-      "port-version": 2
+      "port-version": 3
     },
     "glibmm": {
       "baseline": "2.76.0",

--- a/versions/g-/glib.json
+++ b/versions/g-/glib.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "ac7af1a224e7dfe9c18e629bf8de7798a0cda932",
+      "version": "2.76.3",
+      "port-version": 3
+    },
+    {
       "git-tree": "c4aa04cd3294219ddd4b43c2f02b95c100683156",
       "version": "2.76.3",
       "port-version": 2

